data/remnant/remnant missions.txt

```diff
@@ -1274,6 +1273,30 @@ mission "Remnant: taely intro log"
 
 
 
+mission "Remnant: Chilia Intro Log"
+	invisible
+	source
+		government "Remnant"
```

Comment:
I'd remove the sources. They're pretty redundant with how you're only ever given the "remnant chilia" attribute while in Remnant space.